htmlelements-java/src/main/java/ru/yandex/qatools/htmlelements/loader/decorator/HtmlElementLocatorFactory.java

```diff
@@ -40,6 +44,31 @@ public ElementLocator createLocator(Field field) {
      * @param clazz Class for which locator will be created.
      */
     public ElementLocator createLocator(Class clazz) {
-        return new AjaxElementLocator(searchContext, new HtmlElementClassAnnotationsHandler(clazz));
+        return new AjaxElementLocator(searchContext, getTimeOut(clazz), new HtmlElementClassAnnotationsHandler(clazz));
+    }
+
+    public int getTimeOut(Field field) {
+        if (field.isAnnotationPresent(Timeout.class)) {
+            return field.getDeclaredAnnotation(Timeout.class).value();
```

Comment:
getDelcaredAnnotation(Class<T>) method is not available in java7, please refactor this place to keep java7-compatible